### ISSUE TITLE
crypto_history limit needs to be set with coin_list argument set

### ISSUE THREAD

**Issue by suromeijn** [sstoeckl/crypto2#6]
To start with, amazing package! 

Not sure if intended but when filtering on a single coin in the coin_list argument by first using the crypto_list (within the crypto_history) function) it looks like the limit needs to be set in order for the function to work properly. It gives an error when the limit argument is kept at default (NULL) and when provided it runs. 

Not setting the limit argument and keeping it to NULL (Default) causes error below.

![image](https://user-images.githubusercontent.com/55715314/150687064-8c58c518-9550-4e9f-a247-ae3a3ce9ce0d.png)

https://github.com/sstoeckl/crypto2/blob/3059e48e1017b1ce728627be7e7f1517bb6eba50/R/crypto_history.R#L77

**Comment by sosinage** [comment]
can only agree, fantastic package! 

I want to aquire all crypto data from coinmarketcap, which are older than 2018 to do a time series analysis for my master thesis.

...but I have the same error code when downloading the data. This is my code:
```
library(crypto2)
library(data.table)
library(tidyverse)
library(dplyr)
```
```
# List all active coins that are older than 2018

coins <- as.data.table(crypto_list(only_active=TRUE))
coins <- coins[first_historical_data < "2018-01-01"]
```

```coin_info <- crypto_info(coins,limit=3)```

```coin_hist <- crypto_history(coins, convert = "USD")#, start_date="20100101", end_date="20210105")```

Here appears the error message: "**Error in if (limit == 1) { : argument is of length zero**".

theoretically the next step should be:
```
write.csv(coin_hist, "crypto_data.csv")
coin_hist <- read.csv("crypto_data.csv")
coin_hist$X <- NULL
```

The download includes 187 observations and takes approx. 3 hours - I tried it several times on multiple devices and I am a bit at a loss. Since I'm still new to R, I would be very happy about a suggested solution =)

**Comment by sstoeckl** [comment]
Sorry for the problems, was a stupid error from my side, fixed in ad73c79. Thanks @suromeijn and @sosinage!
